x-pack/plugins/security_solution/public/management/components/endpoint_responder/header_endpoint_info.tsx

```diff
@@ -0,0 +1,102 @@
+import { EndpointAgentAndIsolationStatus } from '../endpoint_agent_and_isolation_status';
+import { getMaybeDate } from '../../../common/components/formatted_date/maybe_date';
+
+interface HeaderEndpointInfoProps {
+  endpointId: string;
+}
+
+export const HeaderEndpointInfo = memo<HeaderEndpointInfoProps>(({ endpointId }) => {
+  const { data: endpointDetails } = useGetEndpointDetails(endpointId, { refetchInterval: 10000 });
+  const { data: endpointPendingActions } = useGetEndpointPendingActionsSummary([endpointId], {
+    refetchInterval: 10000,
+  });
+
+  const pendingIsolationActions = useMemo<
+    Pick<Required<EndpointHostIsolationStatusProps>, 'pendingIsolate' | 'pendingUnIsolate'>
+  >(() => {
+    if (endpointPendingActions?.data.length) {
+      const pendingActions = endpointPendingActions.data[0].pending_actions;
+
+      return {
+        pendingIsolate: pendingActions.isolate ?? 0,
+        pendingUnIsolate: pendingActions.unisolate ?? 0,
+      };
+    }
+    return {
+      pendingIsolate: 0,
+      pendingUnIsolate: 0,
+    };
+  }, [endpointPendingActions?.data]);
+
+  if (!endpointDetails) {
+    return null;
+  }
+
+  if (isFetching) {
+    return <EuiLoadingContent lines={2} />;
+  }
+
+  return (
+    <EuiFlexGroup direction="column" gutterSize="none">
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup alignItems="center" gutterSize="xs">
+          <EuiFlexItem grow={false} className="eui-textTruncate">
+            <EuiToolTip
+              content={endpointDetails.metadata.host.name}
+              anchorClassName="eui-textTruncate"
+            >
+              <EuiText size="s" data-test-subj="responderHeaderEndpointName">
+                <h6 className="eui-textTruncate">
+                  <FormattedMessage
+                    id="xpack.securitySolution.responder.header.endpointName"
+                    defaultMessage="ENDPOINT {name}"
+                    values={{ name: endpointDetails.metadata.host.name }}
+                  />
+                </h6>
+              </EuiText>
+            </EuiToolTip>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EndpointAgentAndIsolationStatus
+              status={endpointDetails.host_status}
+              isIsolated={endpointDetails.metadata.Endpoint.state?.isolation}
+              {...pendingIsolationActions}
+              data-test-subj="responderHeaderEndpointAgentIsolationStatus"
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiText color="subdued" size="s" data-test-subj="responderHeaderLastSeen">
+          <FormattedMessage
+            id="xpack.securitySolution.responder.header.lastSeen"
+            defaultMessage="Last seen {date}"
+            values={{
+              date: (
+                <FormattedRelative
+                  value={moment.utc(getMaybeDate(endpointDetails.metadata['@timestamp'])).toDate()}
```

Comment:
Why do you need to use `moment.utc()` here?

Also - FYI: `FormattedRelative` already handles multiple formats for `value` - `Date | string | number;`

(ref: `node_modules/@types/react-intl/index.d.ts:134`)

Reply:
ah you're right, there was no need for `moment.utc()`